src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/ExceptionRecordCreationTest.java

```diff
@@ -33,18 +36,25 @@
     @Autowired
     private EnvelopeMessager envelopeMessager;
 
+    @Autowired
+    DocumentManagementUploadService dmUploadService;
```

Comment:
could be private. Does PMD not complain?

Reply:
No, PMD didn't complain. Changed it to private